configs/iot-gate-imx8plus_defconfig

```diff
@@ -170,3 +170,5 @@ CONFIG_EFI_CAPSULE_FIRMWARE_FIT=y
 CONFIG_EFI_CAPSULE_AUTHENTICATE=y
 CONFIG_EFI_SECURE_BOOT=y
 CONFIG_ANDROID_BOOT_IMAGE=n
+CONFIG_BOOTDELAY=-2
+CONFIG_SYS_MMC_ENV_DEV=2
```

Comment:
@anedergaard  note this was the missing line that was causing you headache with programming from fastboot earlier on. It would be 0 by default but properly set programatically from somewhere.

Reply:
Yes... I think the boot device scan and setup during normal boot ensures this is set to something meaningful. But we don't perform this until we start to boot so my early reset environment to default after uploading u-boot failed.

Reply:
Or the save of it did... as we relied on the boot logic scripts to correct enviroment for us.